core/src/test/kotlin/io/specmatic/conversions/OpenApiSpecificationTest.kt

```diff
@@ -8864,6 +8864,58 @@ paths:
         }
     }
 
+    @Nested
+    inner class NegativeScenariosForHeaders {
+        @Test
+        fun `should generate the negative scenarios where the mandatory headers are missing`() {
```

Comment:
We need one more test here showing that this functionality works when there is an example (given an example, we'd still generate a negative scenario for that example where the mandatory header is missing).

Reply:
Addressed.